### PR TITLE
updating the .NET Core docs

### PR DESCRIPTION
**Pull request by jaycdave88**
### What does this PR do?
This PR fixes some misconfigured sections to aid customers. 

### Motivation
Helping customers and Support 

### Preview

https://docs-staging.datadoghq.com/jay.dave/aspnetcore_doc_branch/tracing/setup_overview/setup/dotnet-core/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by zacharycmontoya** [comment]
The link in the sentence: "For a full list of supported libraries, visit the Compatibility Requirements page." is messed up, it should be pointing to the .NET Core Compatibility Requirements page but it's pointing to the "Agent main configuration file" page.

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 1]
```diff
@@ -177,35 +122,68 @@ rem Start application
...
-    - `DD_TRACE_AGENT_PORT`
 
-4. The following environment variables are required to enable automatic instrumentation on Linux:
+[1]: /agent/basic_agent_usage/
+[2]: https://en.wikipedia.org/wiki/Musl
+[3]: https://app.datadoghq.com/apm/traces
+{{% /tab %}}
 
-    <div class="alert alert-info"> 
+{{< /tabs >}}
```
Comment (lucaspimentel) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:170]: Everything below this line shows up outside of the Windows/Linux tabs, but the instructions below are all for Linux:
- Set the environment variables by bash script
- Linux Docker container
- SystemCTL (per service)
- SystemCTL (all services)

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 2]
```diff
@@ -285,13 +263,61 @@ When using `systemctl` to run .NET applications as a service, you can also set e
 
 [1]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
 [2]: https://www.freedesktop.org/software/systemd/man/systemctl.html#set-environment%20VARIABLE=VALUE%E2%80%A6
+
+### Configure the Datadog Agent for APM
```
Comment (lucaspimentel) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:267]: This section appears under "Automatic instrumentation," but it applies to all instrumentation. Note that this issue was already present before this PR.

**Comment by lucaspimentel** [comment]
Closing in favor of #10365 which fixes my comments above.

**Pull request closed without merging**